### ISSUE TITLE
Hugchat skill is not working for me anymore?

### ISSUE THREAD

**Issue by Dante1975X** [femelo/skill-ovos-fallback-hugchat#1]
Hugchat skill was working perfect for me, but today I noticed, that it is not working anymore? Ovos is always answering with "error". 

In the ovos core log I found this error line:

`2024-08-21 12:30:16.041 - skill-ovos-fallback-hugchat.femelo - ERROR - ChatBot.query() got an unexpected keyword argument 'web_search'`

**Comment by femelo** [comment]
Thanks for the report, @Dante1975X. The Hugging Chat API is constantly changing, I will try to solve this asap.

**Comment by femelo** [comment]
The problem, as expected, was on HugChat's side. Can you please uninstall the corresponding `hugchat`, the solver plugin and the skill and reinstall the skill?

```
python3 -m pip uninstall  --yes hugchat ovos-solver-hugchat-persona-plugin skill-ovos-fallback-hugchat
```

```
python3 -m pip install git+https://github.com/femelo/skill-ovos-fallback-hugchat.git
```

**Comment by femelo** [comment]
Solved by the PR [#1]( https://github.com/femelo/ovos-solver-plugin-hugchat-persona/pull/1) for the corresponding plugin.
